docs/src/components/components/panel/panel.html

```diff
@@ -0,0 +1,22 @@
+<section class="panel">
+  <header class="panel-header">
+    <h2 class="panel-title">Panel title</h2>
+    <div class="panel-header-content">
+      <span>Panel data point</span>
+      <button>Panel action</button>
+    </div>
+  </header>
+
+  <section class="panel-documentation panel-documentation-desc">
+    <p>Use this marketplace to create service instances for
+      spaces in this org. Then bind service instances to apps using the command line.
+      <a href="https://docs.cloud.gov/apps/managed-services/">Learn about using
+      service instances and marketplaces</a>.
+    </p>
+  </section>
```

Comment:
Mostly just the DRY arguments: reduces the risk of changing in one place and forgetting to change it in another.

But also, in the view tab, fractal will also [link to the sub-component](https://styleguide.fr.cloud.gov/components/detail/trigger.html) which is helpful for docs.

![screenshot from 2017-01-30 16-51-12](https://cloud.githubusercontent.com/assets/509703/22448203/839fb10e-e70c-11e6-85d9-adee8b6d4f91.png)

idk, this is a funny component where it is its own component by our definitions, but `panel-documentation` would never occur outside of a panel, would it?